src/ResourceManager/DataLakeAnalytics/AzureRM.DataLakeAnalytics.psd1

```diff
@@ -54,8 +54,13 @@ CLRVersion = '4.0'
 RequiredModules = @(@{ModuleName = 'AzureRM.Profile'; ModuleVersion = '5.3.0'; })
```

Comment:
This file is actually located inside src/ResourceManager/DataLakeAnalytics/Commands.DataLakeAnalytics in the master branch....is there a reason why it was mode one directory above to DataLakeAnalytics?